connectors/es/sink.py

```diff
@@ -673,9 +673,8 @@ def __init__(self, elastic_config, logger_=None):
     async def prepare_content_index(self, index, language_code=None):
         """Creates the index, given a mapping if it does not exists."""
         if not index.startswith("search-"):
-            raise ContentIndexNameInvalid(
-                'Index name {index} is invalid. Index name must start with "search-"'
-            )
+            msg = 'Index name {index} is invalid. Index name must start with "search-"'
```

Comment:
Apparently this was always meant to be an f-string!

```suggestion
            msg = f"Index name {index} is invalid. Index name must start with 'search-'"
```

Reply:
Ha good catch!